src/scripts/main.js

```diff
@@ -1,3 +1,25 @@
 'use strict';
 
-// write your code here
+const populationList = document.querySelectorAll('.population');
+const totalInner = document.querySelector('.total-population');
+const averageInner = document.querySelector('.average-population');
+
+function getTotal() {
+  let sum = 0;
+
+  for (let i = 0; i < populationList.length; i++) {
+    const number = +populationList[i].textContent.replace(/,/g, '');
+
+    sum += number;
+  }
+  totalInner.innerHTML = sum.toLocaleString('en-US');
```

Comment:
Since we are not dealing with HTML content, it is more appropriate to use `textContent` instead of `innerHtml` for better performance and security.